node/resolvers/Mutations/Users.ts

```diff
@@ -170,8 +170,10 @@ const Users = {
         throw new GraphQLError('organization-data-not-found')
       }
 
-      const roleSlug = await storefrontPermissions.getRole(user.roleId)?.data
-        ?.getRole?.slug
+      const roleSlug = await storefrontPermissionsClient.getRole(user.roleId)
+        .then((result: any) => {
+          return result?.data?.getRole?.slug
+        })
```

Comment:
Thanks, can you just update to avoid using `.then`, and just use await instead? It would be something like this:

```suggestion
      const result = await storefrontPermissionsClient.getRole(user.roleId)
      const roleSlug = result?.data?.getRole?.slug
```